modules/users/client/config/users.client.routes.js

```diff
@@ -358,6 +358,38 @@
           pageTitle: 'Remove profile'
         }
       });
+
+    if (AppConfig.appEnv !== 'production') {
```

Comment:
I'd really like to replace this with the `appSettings.referencesEnabled`, but I don't know how to provide it in here. I tried, failed and got frustrated.

@simison Do you perhaps know how to provide `appSettings` or `SettingsFactory` here?

Reply:
Does adding it to `UsersRoutes` -function work?

```
function UsersRoutes($stateProvider, SettingsService) {
  var settings = SettingsService.get();
```

Reply:
It is failing with error (Firefox) (see the line 3):
```
Error: [$injector:modulerr] Failed to instantiate module trustroots due to:
[$injector:modulerr] Failed to instantiate module users due to:
[$injector:unpr] Unknown provider: SettingsService
http://errors.angularjs.org/1.5.11/$injector/unpr?p0=SettingsService
minErr/<@webpack:///./node_modules/angular/angular.js?:68:12
createInjector/providerCache.$injector<@webpack:///./node_modules/angular/angular.js?:4563:19
getService@webpack:///./node_modules/angular/angular.js?:4716:32
injectionArgs@webpack:///./node_modules/angular/angular.js?:4741:58
invoke@webpack:///./node_modules/angular/angular.js?:4763:18
runInvokeQueue@webpack:///./node_modules/angular/angular.js?:4663:11
...blabla
```

Reply:
Fixing this will require understanding of inner workings of angularjs and meanjs and I'm not motivated to learn it deeply. I'd rather switch things to React asap.
So unless it's easy, maybe it can stay as it is? When the references are finished, this will be removed anyways. Probably not worth the effort, is it?